### PR TITLE
#8: added recursive to uninstall directory resource

### PR DESCRIPTION
**Pull request by bbaugher**

### CODE REVIEW

File: providers/default.rb [hunk 1]
```diff
@@ -234,6 +234,7 @@
 
   directory "#{new_resource.base_dir}/#{new_resource.instance_name}" do
```
Comment (noslowerdna) [inline providers/default.rb:241]: is something validating that base_dir & instance_name aren't empty strings?

**Comment by noslowerdna** [comment]
+1

File: providers/default.rb (hunk 1 above)
Comment (bbaugher) [inline providers/default.rb:241]: No Chef will only check that they are Strings. Added that check here, https://github.com/cerner/cerner_tomcat/pull/10/commits/ed2794950ab15d8cd12ca5818c567ccca4960e08

Comment (noslowerdna) [inline providers/default.rb:241]: Looks good.

**Comment by dhondgepooja** [comment]
+1

**Comment by bbaugher** [comment]
Got enough +1s, merging

**Pull request merged**